### PR TITLE
Update platform requirements and mention Bouncy Castle

### PR DESCRIPTION
**Pull request by seanf**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/zanata/zanata-server/pull/879" target="_blank">ReviewNinja

**Comment by seanf** [comment]
@zanata-jenkins test this please

File: docs/user-guide/system-admin/configuration/authentication.md [hunk 1]
```diff
@@ -219,4 +222,25 @@ It's possible to configure Zanata to use a single pre-defined Open Id authentica
...
       ...
-```
\ No newline at end of file
+```
+
+### Upgrading the Java security provider (for OpenID on Java 1.7)
+
+Java 1.7 is unable to connect to some websites, such as the [Fedora OpenID provider](https://id.fedoraproject.org/).  See [Fedora bug 1163501](https://bugzilla.redhat.com/show_bug.cgi?id=1163501) for details.
+
+**If you are have the option, you will probably find it easier to use Java 1.8 for your server.**
```
Comment (davidmason) [inline docs/user-guide/system-admin/configuration/authentication.md]: Remove "are"

**Comment by definite** [comment]
Note that for java-1.8.0-openjdk-1.8.0.45-35.b13.el6.x86_64, we can do either:
1. remove the sunec.jar
2. Add another line like in standalone.conf

     JAVA_OPTS="$JAVA_OPTS  -Dcom.sun.net.ssl.enableECC=false"

See https://bugzilla.redhat.com/show_bug.cgi?id=1167153

**Comment by carlosmunoz** [comment]
@seanf Can we add something about @definite's findings to the docs?

**Comment by seanf** [comment]
@carlosmunoz done!

**Comment by carlosmunoz** [comment]
:+1:

**Comment by seanf** [comment]
Squash-merged as cf62639 and e2036b5.

**Pull request closed without merging**